src/components/AmountEditable.tsx

```diff
@@ -187,13 +250,31 @@ export const AmountEditable: ParentComponent<{
         props.initialAmountSats || "0"
     );
     const [localFiat, setLocalFiat] = createSignal(
-        satsToUsd(
+        satsToFiat(
             state.price,
             parseInt(props.initialAmountSats || "0") || 0,
-            false
+            state.fiat
         )
     );
 
+    const fixedAmount = (multiplier: number, label: boolean) => {
+        //This returns a stringified number based on the price range of the chosen currency as compared to BTC
```

Comment:
I appreciate comments like this but they still require me to run a compiler in my head. even one example below this comment would be helpful for reading this